### DISCUSSION TITLE
A Feedback button in the platform should point to github discussions

### DISCUSSION THREAD

**Discussion by jtwaleson** [darklang/dark#4553] [Suggestions]
Right now I had to ask on Discord where to post feedback. Which is here apparently :) It would be great if there was a "feedback" button in the product.

**Comment by pbiggar** [comment]
![Screenshot 2023-01-27 at 2 37 05 PM](https://user-images.githubusercontent.com/181762/215181615-7b0aa994-f337-4775-bfcf-479d25e85b53.png)

This is implemented!

**Reply by pbiggar** [reply]
"Post feedback"
